Crash when trying to log any cache

### Describe your problem!

Today I needed to log three different OC caches. The problem is not clearly reproducible but somehow happening on first try but not on consecutive tries.


### How to reproduce?

- Store OC140A1
- Select Menu - Quick offline log - Found (offline logging enabled)
- Open the cache again later
- Select Menu - Log - Log visit (in order to review and finally send the log)


### Actual result after these steps?

App crashes
When trying again it usually works for that cache but shows the same problem for the next OC-cache I tried

```
8-29 21:12:00.877 25794 25794 E AndroidRuntime: FATAL EXCEPTION: main
08-29 21:12:00.877 25794 25794 E AndroidRuntime: Process: cgeo.geocaching, PID: 25794
08-29 21:12:00.877 25794 25794 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setEnabled(boolean)' on a null object reference
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at cgeo.geocaching.log.LogCacheActivity.onLoadFinished(LogCacheActivity.java:154)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at cgeo.geocaching.connector.oc.OkapiLoggingManager.onLoadFinished(OkapiLoggingManager.java:75)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at cgeo.geocaching.connector.oc.OkapiLoggingManager.onLoadFinished(OkapiLoggingManager.java:26)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at androidx.loader.app.LoaderManagerImpl$LoaderObserver.onChanged(LoaderManagerImpl.java:250)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at androidx.lifecycle.LiveData.considerNotify(LiveData.java:133)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:151)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at androidx.lifecycle.LiveData.setValue(LiveData.java:309)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at androidx.lifecycle.MutableLiveData.setValue(MutableLiveData.java:50)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at androidx.loader.app.LoaderManagerImpl$LoaderInfo.setValue(LoaderManagerImpl.java:189)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at androidx.loader.app.LoaderManagerImpl$LoaderInfo.onLoadComplete(LoaderManagerImpl.java:174)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at androidx.loader.content.Loader.deliverResult(Loader.java:132)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at androidx.loader.content.AsyncTaskLoader.dispatchOnLoadComplete(AsyncTaskLoader.java:258)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at androidx.loader.content.AsyncTaskLoader$LoadTask.onPostExecute(AsyncTaskLoader.java:83)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at androidx.loader.content.ModernAsyncTask.finish(ModernAsyncTask.java:490)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at androidx.loader.content.ModernAsyncTask$InternalHandler.handleMessage(ModernAsyncTask.java:507)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8669)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
08-29 21:12:00.877 25794 25794 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
```

### Expected result after these steps?

Normal operation (= opening loggging view)

### Reproducible

Unclear

### c:geo Version

2022.08.29-NB

### System information

```
## System information

c:geo version: 2022.08.29-NB-923f71d

Device:
-------
- Device type: SM-G980F (x1seea, samsung)
- Available processors: 8
- Android version: 12
- Android build: SP1A.210812.016.G980FXXUEFVG5
- Screen resolution: 1080x2280px (384x810dp)
- Pixel density: 2.53125
- System font scale: 0.9 / used scale: 0.9
- Sailfish OS detected: false
- Google Play services: enabled - 22.26.15 (190408-461192076)

Sensor and location:
-------
- Low power mode: inactive
- Compass capabilities: yes
- Rotation vector sensor: present
- Orientation sensor: present
- Magnetometer & Accelerometer sensor: present
- Direction sensor used: rotation vector

Program settings:
-------
- Settings: v9, Count:258
- Set language: de_DE (system default)
- System date format: dd.MM.yy
- Time zone: GMT+02:00
- Debug mode active: no
- Last backup: 9. Juli, 12:30
- Routing mode: Walk
- Live map mode: true
- OSM multi-threading: true / threads: 4
- Map: Germany_oam.osm (Offline)
  - Id: cgeo.geocaching.maps.mapsforge.MapsforgeMapProvider$OfflineMapSource:5437-5937:VectorMaps/Germany_oam.osm.map
  - Atts: Maps: (c)Christian Kernbeis & Michael Bechtold , Themes: (c)Tobias Kuehn, www.openandromaps.org 2012-2022
  - Theme: Elevate.zip:Elevate.xml

Filters:
-------
- Hide waypoints: original visited
- LIVE: Ohne Eigene/Gefundene ([Ohne Eigene/Gefundene:inconclusive=false:advanced=true]AND(type;status:owned_no:found_no))
- OFFLINE: Keine ([:inconclusive=false:advanced=true])

Stored Filters (#5):
- Hamburg: [Hamburg:inconclusive=false:advanced=false]AND(type:USER_DEFINED:VIRTUAL:ADVLAB:MYSTERY:TRADITIONAL:WEBCAM:LETTERBOX:EARTH;status:owned_no:found_no:solved_mystery_yes))
- Mit dem Rad unterwegs: [Mit dem Rad unterwegs:inconclusive=false:advanced=false]AND(type:VIRTUAL:ADVLAB:WEBCAM:TRADITIONAL:EARTH;difficulty_terrain:d=-:d=-:t=1.0:t=4.0;status:owned_no:found_no))
- Ohne Eigene/Gefundene: [Ohne Eigene/Gefundene:inconclusive=false:advanced=true]AND(type;status:owned_no:found_no))
- Ohne Eigene/Gefundene, nur gelöste Mysteries: [Ohne Eigene/Gefundene, nur gelöste Mysteries:inconclusive=false:advanced=true]AND(status:owned_no:found_no:solved_mystery_yes))
- test: [test:inconclusive=false:advanced=true]AND(favorites:-:-:percentage=true))

Services:
-------
- Geocaching sites enabled:
   geocaching.com: Logged in (Anmeldung OK) / PREMIUM
   Geocaching.com Adventure Lab
   opencaching.de: Logged in (Anmeldung OK)
- Geocaching.com date format: d.M.yyyy
- Last login error on geocaching.com: communication error (2022-08-17 12-53)
- Last successful login on geocaching.com: 2022-08-29 21-12
- Routing: internal / BRouter installed: true
- Installed c:geo plugins: contacts

Permissions & paths:
-------
- Fine location permission: granted
- Write external storage permission: granted
- System internal c:geo dir: /data/user/0/cgeo.geocaching (38,9 GB free) v3 internal isDir(8 entries)
- Legacy User storage c:geo dir: /data/user/0/cgeo.geocaching (38,9 GB free) v3 internal isDir(8 entries)
- Geocache data: /storage/5437-5937/Android/data/cgeo.geocaching/files/GeocacheData (368,6 GB free) v3 external removable isDir(1200 entries)
- Internal theme sync (is turned off): /data/user/0/cgeo.geocaching/MapThemeData (38,9 GB free) v3 internal isDir(0 entries)
- Public Folders: #12
  - BASE: /cgeo (User-Defined)[/cgeo[DOCUMENT#0:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo, Av:true, files:>=7, dirs:>=17, totalFileSize:>=1,5 MB, free space: 38,9 GB, files on device: 13511675)
  - OFFLINE_MAPS: Lars_512GB/VectorMaps (User-Defined)[Lars_512GB/VectorMaps[DOCUMENT#0:p-content://com.android.externalstorage.documents/tree/5437-5937%3AVectorMaps::]] (Uri: content://com.android.externalstorage.documents/tree/5437-5937%3AVectorMaps/document/5437-5937%3AVectorMaps, Av:true, files:>=116, dirs:>=10, totalFileSize:>=13,8 GB, free space: 368,6 GB, files on device: 0)
  - OFFLINE_MAP_THEMES: Lars_512GB/VectorMaps/_themes (User-Defined)[Lars_512GB/VectorMaps/_themes[DOCUMENT#0:p-content://com.android.externalstorage.documents/tree/5437-5937%3AVectorMaps%2F_themes::]] (Uri: content://com.android.externalstorage.documents/tree/5437-5937%3AVectorMaps%2F_themes/document/5437-5937%3AVectorMaps%2F_themes, Av:true, files:>=303, dirs:>=8, totalFileSize:>=3,1 MB, free space: 368,6 GB, files on device: 0)
  - OFFLINE_MAP_SHADING: Lars_512GB/VectorMaps/_hgt (Default)[Lars_512GB/VectorMaps/_hgt[PERSISTABLE_FOLDER(OFFLINE_MAPS)#1:p-content://com.android.externalstorage.documents/tree/5437-5937%3AVectorMaps::/_hgt]] (Uri: content://com.android.externalstorage.documents/tree/5437-5937%3AVectorMaps/document/5437-5937%3AVectorMaps%2F_hgt, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 368,6 GB, files on device: 0)
  - LOGFILES: /cgeo/logfiles (Default)[/cgeo/logfiles[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/logfiles]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Flogfiles, Av:true, files:28, dirs:0, totalFileSize:8,2 MB, free space: 38,9 GB, files on device: 13511675)
  - GPX: /cgeo/gpx (Default)[/cgeo/gpx[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/gpx]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Fgpx, Av:true, files:36, dirs:0, totalFileSize:42,9 MB, free space: 38,9 GB, files on device: 13511675)
  - BACKUP: /cgeo/backup (Default)[/cgeo/backup[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/backup]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Fbackup, Av:true, files:>=10, dirs:>=8, totalFileSize:>=161,4 MB, free space: 38,9 GB, files on device: 13511675)
  - FIELD_NOTES: /cgeo/field-notes (Default)[/cgeo/field-notes[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/field-notes]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Ffield-notes, Av:true, files:187, dirs:0, totalFileSize:357,5 KB, free space: 38,9 GB, files on device: 13511675)
  - SPOILER_IMAGES: /cgeo/GeocachePhotos (Default)[/cgeo/GeocachePhotos[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/GeocachePhotos]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2FGeocachePhotos, Av:true, files:>=1, dirs:>=12, totalFileSize:>=658,8 KB, free space: 38,9 GB, files on device: 13511675)
  - ROUTING_BASE: /cgeo/routing (Default)[/cgeo/routing[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/routing]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Frouting, Av:true, files:11, dirs:1, totalFileSize:134,0 MB, free space: 38,9 GB, files on device: 13511675)
  - ROUTING_TILES: /cgeo/routing/segments4 (User-Defined)[/cgeo/routing/segments4[DOCUMENT#0:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Frouting%2Fsegments4::]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Frouting%2Fsegments4/document/primary%3Acgeo%2Frouting%2Fsegments4, Av:true, files:1, dirs:0, totalFileSize:133,9 MB, free space: 38,9 GB, files on device: 13511675)
  - TEST_FOLDER: [Legacy]/data/user/0/cgeo.geocaching/files/unittest (Default)[/data/user/0/cgeo.geocaching/files/unittest[FILE#1:p-file:///data/user/0/cgeo.geocaching/files::/unittest]] (Uri: file:///data/user/0/cgeo.geocaching/files/unittest, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 38,9 GB, files on device: -1)
- Map render theme path: Elevate.zip:Elevate.xml
- PersistedDocumentUris: #1
- UNUSED_TRACK: content://com.android.providers.downloads.documents/document/1515
- Persisted Uri Permissions: #4
  - content://com.android.externalstorage.documents/tree/5437-5937%3AVectorMaps (11. Aug., 23:42):RW
  - content://com.android.externalstorage.documents/tree/primary%3Acgeo (22. Aug. 2021, 20:28):RW
  - content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Frouting%2Fsegments4 (24. Aug., 17:32):RW
  - content://com.android.externalstorage.documents/tree/5437-5937%3AVectorMaps%2F_themes (18. März, 11:05):RW
- Database: /data/user/0/cgeo.geocaching/databases/data (v99, Size:32,5 MB) on system internal storage

--- End of system information ---
```


### Additional Information

_No response_

Seems not to be restricted to OC.
Nightly users report the same also for GC caches, e.g.:
```
08-29 23:54:24.699 18260 18260 W cgeo    : [main] UncaughtException
08-29 23:54:24.699 18260 18260 W cgeo    : java.lang.RuntimeException: Unable to start activity ComponentInfo{cgeo.geocaching/cgeo.geocaching.log.LogCacheActivity}: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setEnabled(boolean)' on a null object reference
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4035)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4201)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.os.Handler.dispatchMessage(Handler.java:106)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.os.Looper.loopOnce(Looper.java:226)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.os.Looper.loop(Looper.java:313)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.app.ActivityThread.main(ActivityThread.java:8669)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at java.lang.reflect.Method.invoke(Native Method)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
08-29 23:54:24.699 18260 18260 W cgeo    : Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setEnabled(boolean)' on a null object reference
08-29 23:54:24.699 18260 18260 W cgeo    : 	at cgeo.geocaching.log.LogCacheActivity.showProgress(LogCacheActivity.java:452)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at cgeo.geocaching.log.LogCacheActivity.onLoadStarted(LogCacheActivity.java:127)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at cgeo.geocaching.connector.gc.GCLoggingManager.onCreateLoader(GCLoggingManager.java:67)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at androidx.loader.app.LoaderManagerImpl.createAndInstallLoader(LoaderManagerImpl.java:383)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at androidx.loader.app.LoaderManagerImpl.initLoader(LoaderManagerImpl.java:421)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at cgeo.geocaching.connector.gc.GCLoggingManager.init(GCLoggingManager.java:160)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at cgeo.geocaching.log.LogCacheActivity.onCreate(LogCacheActivity.java:261)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.app.Activity.performCreate(Activity.java:8290)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.app.Activity.performCreate(Activity.java:8270)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1329)
08-29 23:54:24.699 18260 18260 W cgeo    : 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4009)
08-29 23:54:24.699 18260 18260 W cgeo    : 	... 12 more
```

System info:

```
## System information

c:geo version: 2022.08.30-NB-d1040ce

Device:
-------
- Device type: SM-A226B (a22xnseea, samsung)
- Available processors: 8
- Android version: 12
- Android build: SP1A.210812.016.A226BXXU4BVF7
- Screen resolution: 1080x2151px (360x714dp)
- Pixel density: 3.0
- System font scale: 1.0 / used scale: 1.0
- Sailfish OS detected: false
- Google Play services: enabled - 22.26.15 (190400-461192076)

Sensor and location:
-------
- Low power mode: inactive
- Compass capabilities: yes
- Rotation vector sensor: present
- Orientation sensor: present
- Magnetometer & Accelerometer sensor: present
- Direction sensor used: rotation vector

Program settings:
-------
- Settings: v9, Count:151
- Set language: de_DE (system default)
- System date format: [dd.MM.yy](http://dd.mm.yy/)
- Time zone: MDT
- Debug mode active: no
- Last backup: 18. Juli, 17:01
- Routing mode: Walk
- Live map mode: true
- OSM multi-threading: false / threads: 1
- Map: Us-west (Offline)
- Id: [cgeo.geocaching.maps.mapsforge.MapsforgeMapProvider](http://cgeo.geocaching.maps.mapsforge.mapsforgemapprovider/)$OfflineMapSource:primary:Cgeo/maps/[us-west.map](http://us-west.map/)
- Atts: Map data (c) OpenStreetMap contributors
- Theme: none

Filters:
-------
- Hide waypoints: -
- LIVE: Cache-Typ:Earth,Status:Gefunden=Nein ([:inconclusive=false:advanced=true]AND(type:EARTH;difficulty_terrain:d=-:d=-:t=-:t=-;size;status:found_no))
- OFFLINE: Keine ([:inconclusive=false:advanced=true]AND(size;difficulty_terrain:d=-:d=-:t=-:t=-;type;status))

Stored Filters (#0):

Services:
-------
- Geocaching sites enabled:
[geocaching.com](http://geocaching.com/): Logged in (Anmeldung OK) / PREMIUM
[Geocaching.com](http://geocaching.com/) Adventure Lab
- [Geocaching.com](http://geocaching.com/) date format: [dd.MM.yyyy](http://dd.mm.yyyy/)
- Last login error on [geocaching.com](http://geocaching.com/): communication error (2022-08-29 12-04)
- Last successful login on [geocaching.com](http://geocaching.com/): 2022-08-29 23-54
- Routing: internal / BRouter installed: false
- Installed c:geo plugins: none

Permissions & paths:
-------
- Fine location permission: granted
- Write external storage permission: granted
- System internal c:geo dir: /data/user/0/[cgeo.geocaching](http://cgeo.geocaching/) (14,8 GB free) v3 internal isDir(19 entries)
- Legacy User storage c:geo dir: /storage/emulated/0/cgeo (14,8 GB free) v3 external non-removable isDir(7 entries)
- Geocache data: /storage/emulated/0/Android/data/[cgeo.geocaching/files/GeocacheData](http://cgeo.geocaching/files/GeocacheData) (14,8 GB free) v3 external non-removable isDir(1668 entries)
- Internal theme sync (is turned off): /data/user/0/[cgeo.geocaching/MapThemeData](http://cgeo.geocaching/MapThemeData) (14,8 GB free) v3 internal isDir(0 entries)
- Public Folders: #12
- BASE: /Cgeo (User-Defined)[/Cgeo[DOCUMENT#0:p-content://[com.android.externalstorage.documents/tree/primary%3ACgeo](http://com.android.externalstorage.documents/tree/primary%3ACgeo)::]] (Uri: content://[com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo](http://com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo), Av:true, files:>=37, dirs:>=9, totalFileSize:>=1,6 GB, free space: 14,8 GB, files on device: 12716544)
- OFFLINE_MAPS: /Cgeo/maps (Default)[/Cgeo/maps[PERSISTABLE_FOLDER(BASE)#1:p-content://[com.android.externalstorage.documents/tree/primary%3ACgeo::/maps](https://cgeosupport.droescher.eu/scp/com.android.externalstorage.documents/tree/primary%3ACgeo::/maps)]] (Uri: content://[com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Fmaps](http://com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Fmaps), Av:true, files:5, dirs:2, totalFileSize:1,6 GB, free space: 14,8 GB, files on device: 12716544)
- OFFLINE_MAP_THEMES: /Cgeo/maps/_themes (Default)[/Cgeo/maps/_themes[PERSISTABLE_FOLDER(OFFLINE_MAPS)#1:p-content://[com.android.externalstorage.documents/tree/primary%3ACgeo::/maps/_themes](https://cgeosupport.droescher.eu/scp/com.android.externalstorage.documents/tree/primary%3ACgeo::/maps/_themes)]] (Uri: content://[com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Fmaps%2F_themes](http://com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Fmaps%2F_themes), Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 14,8 GB, files on device: 12716544)
- OFFLINE_MAP_SHADING: /Cgeo/maps/_hgt (Default)[/Cgeo/maps/_hgt[PERSISTABLE_FOLDER(OFFLINE_MAPS)#1:p-content://[com.android.externalstorage.documents/tree/primary%3ACgeo::/maps/_hgt](https://cgeosupport.droescher.eu/scp/com.android.externalstorage.documents/tree/primary%3ACgeo::/maps/_hgt)]] (Uri: content://[com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Fmaps%2F_hgt](http://com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Fmaps%2F_hgt), Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 14,8 GB, files on device: 12716544)
- LOGFILES: /Cgeo/logfiles (Default)[/Cgeo/logfiles[PERSISTABLE_FOLDER(BASE)#1:p-content://[com.android.externalstorage.documents/tree/primary%3ACgeo::/logfiles](https://cgeosupport.droescher.eu/scp/com.android.externalstorage.documents/tree/primary%3ACgeo::/logfiles)]] (Uri: content://[com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Flogfiles](http://com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Flogfiles), Av:true, files:7, dirs:0, totalFileSize:2,5 MB, free space: 14,8 GB, files on device: 12716544)
- GPX: /Cgeo/gpx (Default)[/Cgeo/gpx[PERSISTABLE_FOLDER(BASE)#1:p-content://[com.android.externalstorage.documents/tree/primary%3ACgeo::/gpx](https://cgeosupport.droescher.eu/scp/com.android.externalstorage.documents/tree/primary%3ACgeo::/gpx)]] (Uri: content://[com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Fgpx](http://com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Fgpx), Av:true, files:25, dirs:0, totalFileSize:14,5 MB, free space: 14,8 GB, files on device: 12716544)
- BACKUP: /Cgeo/backup (Default)[/Cgeo/backup[PERSISTABLE_FOLDER(BASE)#1:p-content://[com.android.externalstorage.documents/tree/primary%3ACgeo::/backup](https://cgeosupport.droescher.eu/scp/com.android.externalstorage.documents/tree/primary%3ACgeo::/backup)]] (Uri: content://[com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Fbackup](http://com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Fbackup), Av:true, files:>=12, dirs:>=5, totalFileSize:>=556,1 MB, free space: 14,8 GB, files on device: 12716544)
- FIELD_NOTES: /Cgeo/field-notes (Default)[/Cgeo/field-notes[PERSISTABLE_FOLDER(BASE)#1:p-content://[com.android.externalstorage.documents/tree/primary%3ACgeo::/field-notes](https://cgeosupport.droescher.eu/scp/com.android.externalstorage.documents/tree/primary%3ACgeo::/field-notes)]] (Uri: content://[com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Ffield-notes](http://com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Ffield-notes), Av:true, files:16, dirs:0, totalFileSize:86,6 KB, free space: 14,8 GB, files on device: 12716544)
- SPOILER_IMAGES: /Cgeo/GeocachePhotos (Default)[/Cgeo/GeocachePhotos[PERSISTABLE_FOLDER(BASE)#1:p-content://[com.android.externalstorage.documents/tree/primary%3ACgeo::/GeocachePhotos](https://cgeosupport.droescher.eu/scp/com.android.externalstorage.documents/tree/primary%3ACgeo::/GeocachePhotos)]] (Uri: content://[com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2FGeocachePhotos](http://com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2FGeocachePhotos), Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 14,8 GB, files on device: 12716544)
- ROUTING_BASE: /Cgeo/routing (Default)[/Cgeo/routing[PERSISTABLE_FOLDER(BASE)#1:p-content://[com.android.externalstorage.documents/tree/primary%3ACgeo::/routing](https://cgeosupport.droescher.eu/scp/com.android.externalstorage.documents/tree/primary%3ACgeo::/routing)]] (Uri: content://[com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Frouting](http://com.android.externalstorage.documents/tree/primary%3ACgeo/document/primary%3ACgeo%2Frouting), Av:true, files:20, dirs:1, totalFileSize:581,7 MB, free space: 14,8 GB, files on device: 12716544)
- ROUTING_TILES: /Cgeo/routing/segments4 (User-Defined)[/Cgeo/routing/segments4[DOCUMENT#0:p-content://[com.android.externalstorage.documents/tree/primary%3ACgeo%2Frouting%2Fsegments4](http://com.android.externalstorage.documents/tree/primary%3ACgeo%2Frouting%2Fsegments4)::]] (Uri: content://[com.android.externalstorage.documents/tree/primary%3ACgeo%2Frouting%2Fsegments4/document/primary%3ACgeo%2Frouting%2Fsegments4](http://com.android.externalstorage.documents/tree/primary%3ACgeo%2Frouting%2Fsegments4/document/primary%3ACgeo%2Frouting%2Fsegments4), Av:true, files:12, dirs:0, totalFileSize:581,6 MB, free space: 14,8 GB, files on device: 12716544)
- TEST_FOLDER: [Legacy]/data/user/0/[cgeo.geocaching/files/unittest](http://cgeo.geocaching/files/unittest) (Default)[/data/user/0/[cgeo.geocaching/files/unittest](http://cgeo.geocaching/files/unittest)[FILE#1:p-file:///data/user/0/[cgeo.geocaching/files::/unittest](https://cgeosupport.droescher.eu/scp/cgeo.geocaching/files::/unittest)]] (Uri: file:///data/user/0/[cgeo.geocaching/files/unittest](http://cgeo.geocaching/files/unittest), Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 14,8 GB, files on device: -1)
- Map render theme path:
- PersistedDocumentUris: #1
- UNUSED_TRACK: null
- Persisted Uri Permissions: #2
- content://[com.android.externalstorage.documents/tree/primary%3ACgeo%2Frouting%2Fsegments4](http://com.android.externalstorage.documents/tree/primary%3ACgeo%2Frouting%2Fsegments4) (8. Nov. 2021, 23:42):RW
- content://[com.android.externalstorage.documents/tree/primary%3ACgeo](http://com.android.externalstorage.documents/tree/primary%3ACgeo) (23. Aug. 2021, 13:51):RW
- Database: /data/user/0/[cgeo.geocaching/databases/data](http://cgeo.geocaching/databases/data) (v99, Size:68,0 MB) on system internal storage

--- End of system information ---
```

Support ticket 826416

--- a/main/res/menu/abstract_logging_activity.xml
+++ b/main/res/menu/abstract_logging_activity.xml
@@ -4,7 +4,6 @@
 
     <item
         android:id="@+id/menu_send"
-        android:enabled="false"
         android:icon="@drawable/ic_menu_send"
         android:title="@string/send"
         app:showAsAction="ifRoom|withText">

--- a/main/res/values/strings.xml
+++ b/main/res/values/strings.xml
@@ -143,6 +143,7 @@
     <string name="log_posting_generic_trackable">Posting %1$s %2$d/%3$d</string>
     <string name="log_clear">Clear</string>
     <string name="log_templates">Found it: Log template…</string>
+    <string name="log_post_not_possible">Download of data in progress…\nPlease try again in a few seconds or check your internet connection.</string>
     <string name="log_date_future_not_allowed">Future log dates are not allowed.</string>
     <string name="log_add">Add</string>
     <string name="log_repeat">Repeat last log</string>

--- a/main/src/cgeo/geocaching/log/LogCacheActivity.java
+++ b/main/src/cgeo/geocaching/log/LogCacheActivity.java
@@ -99,7 +99,7 @@ public class LogCacheActivity extends AbstractLoggingActivity {
     private final TextSpinner<LogType> logType = new TextSpinner<>();
     private final DateTimeEditor date = new DateTimeEditor();
 
-    private MenuItem sendButton;
+    private boolean readyToPost = false;
     private final TextSpinner<ReportProblemType> reportProblem = new TextSpinner<>();
     private final TextSpinner<LogTypeTrackable> trackableActionsChangeAll = new TextSpinner<>();
 
@@ -449,7 +449,7 @@ public class LogCacheActivity extends AbstractLoggingActivity {
     }
 
     private void showProgress(final boolean loading) {
-        sendButton.setEnabled(!loading);
+        readyToPost = !loading;
         binding.progressBar.setVisibility(loading ? View.VISIBLE : View.GONE);
     }
 
@@ -543,6 +543,10 @@ public class LogCacheActivity extends AbstractLoggingActivity {
     }
 
     private void sendLogAndConfirm() {
+        if (!readyToPost) {
+            SimpleDialog.of(this).setMessage(R.string.log_post_not_possible).show();
+            return;
+        }
         if (CalendarUtils.isFuture(date.getCalendar())) {
             SimpleDialog.of(this).setMessage(R.string.log_date_future_not_allowed).show();
             return;
@@ -565,7 +569,6 @@ public class LogCacheActivity extends AbstractLoggingActivity {
     @Override
     public boolean onCreateOptionsMenu(final Menu menu) {
         super.onCreateOptionsMenu(menu);
-        sendButton = menu.findItem(R.id.menu_send);
         menu.findItem(R.id.menu_image).setVisible(cache.supportsLogImages());
         menu.findItem(R.id.save).setVisible(true);
         menu.findItem(R.id.clear).setVisible(true);

--- a/main/src/cgeo/geocaching/log/LogTrackableActivity.java
+++ b/main/src/cgeo/geocaching/log/LogTrackableActivity.java
@@ -71,7 +71,7 @@ public class LogTrackableActivity extends AbstractLoggingActivity implements Coo
     /**
      * As long as we still fetch the current state of the trackable from the Internet, the user cannot yet send a log.
      */
-    private MenuItem sendButton;
+    private boolean readyToPost = true;
     private final DateTimeEditor date = new DateTimeEditor();
     private LogTypeTrackable typeSelected = LogTypeTrackable.getById(Settings.getTrackableAction());
     private Trackable trackable;
@@ -349,7 +349,7 @@ public class LogTrackableActivity extends AbstractLoggingActivity implements Coo
     }
 
     private void showProgress(final boolean loading) {
-        sendButton.setEnabled(!loading);
+        readyToPost = !loading;
         binding.progressBar.setVisibility(loading ? View.VISIBLE : View.GONE);
     }
 
@@ -520,6 +520,12 @@ public class LogTrackableActivity extends AbstractLoggingActivity implements Coo
      * Do form validation then post the Log
      */
     private void sendLog() {
+        // Can logging?
+        if (!readyToPost) {
+            showToast(res.getString(R.string.log_post_not_possible));
+            return;
+        }
+
         // Check Tracking Code existence
         if (loggingManager.isTrackingCodeNeededToPostNote() && binding.tracking.getText().toString().isEmpty()) {
             showToast(res.getString(R.string.err_log_post_missing_tracking_code));
@@ -556,7 +562,6 @@ public class LogTrackableActivity extends AbstractLoggingActivity implements Coo
     @Override
     public boolean onCreateOptionsMenu(final Menu menu) {
         final boolean result = super.onCreateOptionsMenu(menu);
-        sendButton = menu.findItem(R.id.menu_send);
         for (final LogTemplate template : LogTemplateProvider.getTemplatesWithoutSignature()) {
             if (template.getTemplateString().equals("NUMBER") || template.getTemplateString().equals("ONLINENUM")) {
                 menu.findItem(R.id.menu_templates).getSubMenu().removeItem(template.getItemId());

